### ISSUE TITLE
[showstopper] MP distancia4 bugged, SP is working

### ISSUE THREAD

**Issue by StefArma** [A3Antistasi/Antistasi-WotP#13]
*Version:* 0.11.1A 

*Mods:* CBA, TFAR, Ace no Medical

*Environment*: MP LAN and  MP dedi.

*.rpt attatched?* 
if (lados getVariable [_marcador,sideUnknown>
 6:55:37   Error position: <lados getVariable [_marcador,sideUnknown>
 6:55:37   Error Undefined variable in expression: lados
 6:55:37 File mpmissions\__cur_mp.Tanoa\distancias4.sqf, line 56
 6:55:37 Error in expression <ionMRK = getMarkerPos (_marcador);

if (lados getVariable [_marcador,sideUnknown>
 6:55:37   Error position: <lados getVariable [_marcador,sideUnknown>
 6:55:37   Error Undefined variable in expression: lados
 6:55:37 File mpmissions\__cur_mp.Tanoa\distancias4.sqf, line 56
 6:55:37 Error in expression <ionMRK = getMarkerPos (_marcador);

if (lados getVariable [_marcador,sideUnknown>
 6:55:37   Error position: <lados getVariable [_marcador,sideUnknown>
 6:55:37   Error Undefined variable in expression: lados
 6:55:37 File mpmissions\__cur_mp.Tanoa\distancias4.sqf, line 56
 6:55:37 Error in expression <ionMRK = getMarkerPos (_marcador);

if (lados getVariable [_marcador,sideUnknown>
 6:55:37   Error position: <lados getVariable [_marcador,sideUnknown>
 6:55:37   Error Undefined variable in expression: lados
 6:55:37 File mpmissions\__cur_mp.Tanoa\distancias4.sqf, line 56
 6:55:37 Error in expression <ionMRK = getMarkerPos (_marcador);

*have you edited the missionfile?*: No

***Issue:*** nothing spawn in MP

**Comment by Barbolani77** [comment]
fuck, the only reason I can find to this is: a. someone deleteVehicle lados, b. you are using old misión.sqm + new scripts on which the GL named lados is not there.

lados cannot be undefined as it is a GL placed in the map, and the getVariable [_marcador is bulletproof as has a default value even when undefined.

**Comment by StefArma** [comment]
let me try again, i tested master branch of git

**Comment by StefArma** [comment]
i don't know what went wrong before, now works. I shouldn't test in hangover XD

**Comment by Barbolani77** [comment]
I shouldnt code in hangover :)

**Comment by StefArma** [comment]
I got the problem, basically i was in editor and i switched to the master branch my sqm was still cached in and it didn't synced it

**Comment by Barbolani77** [comment]
soooooo closed!
